src/duplicates-ind1.js

```diff
@@ -28,7 +28,7 @@
 
 'use strict';
 
-export default async function(tagPattern) {
+export default async function (tagPattern) {
```

Comment:
Stash changes to this file. The PR should not modify this validator.

Reply:
fixed